tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_creator.py

```diff
@@ -136,17 +137,30 @@ def get_config(self):
     return config
 
 
+@unique
+class HkvEvictStrategy(IntEnum):
+  LRU = 0
+  LFU = 1
+  EPOCHLRU = 2
+  EPOCHLFU = 3
+  CUSTOMIZED = 4
+
+
 class HkvHashTableConfig(object):
 
   def __init__(self,
                init_capacity=KHkvHashTableInitCapacity,
                max_capacity=KHkvHashTableMaxCapacity,
-               max_hbm_for_values=KHkvHashTableMaxHbmForValuesByBytes):
+               max_hbm_for_values=KHkvHashTableMaxHbmForValuesByBytes,
+               evict_strategy=HkvEvictStrategy.LRU,
+               evict_global_epoch=0):
```

Comment:
The default should be 'IGNORED_GLOBAL_EPOCH' which is `0xffffffffffffffff', pls refer to the code : https://github.com/NVIDIA-Merlin/HierarchicalKV/blob/master/include/merlin/types.cuh#L283

Reply:
> error: no instance of overloaded function "tensorflow::GetNodeAttr" matches the argument list
            argument types are: (const tensorflow::NodeDef, const char [19], tensorflow::uint64 *)

GetNodeAttr does not support getting uint64_t arguments, so 0xFFFFFFFFFFFFFFFF cannot be passed as the default value